bpf/xdp.h

```diff
@@ -24,7 +24,6 @@ static int xdp_ingress(struct xdp_md *ctx OVS_UNUSED)
     /* TODO: see p4c-xdp project */
 #ifdef BPF_ENABLE_IPV6
 	printt("return XDP_PASS\n");
-    return XDP_PASS;
```

Comment:
I moved the XDP_PASS to line 44. So that the following two cases will both return XDP_PASS.
1) All the traffic when enable IPv6
2) non-IPv6 traffic when disable IPv6. 

Currently, we do not return XDP_PASS for case 2.

Reply:
I see, thanks.